### PR TITLE
Retrieve parent keys available to a child section

### PR DESCRIPTION
**Pull request by AT1012**
This PR lets us retrieve a list of parent keys that are
available to the child section.

### CODE REVIEW

File: key_test.go [hunk 1]
```diff
@@ -59,6 +59,13 @@ func Test_Key(t *testing.T) {
 			}
 		})
 
+		Convey("Get parent-keys that are available to the child section", func() {
+			parentKeys := cfg.Section("package.sub").ParentKeys()
+			for _, k := range parentKeys {
+				So(k.Name(), ShouldEqual, fmt.Sprintf("%s", "CLONE_URL"))
```
Comment (Unknwon) [inline key_test.go]: I think you can replace `fmt.Sprintf("%s", "CLONE_URL")` with `"CLONE_URL"`, plain text string does not need format.

**Comment by Unknwon** [comment]
Other than that, LGTM.

File: key_test.go (hunk 1 above)
Comment (AT1012) [inline key_test.go]: Fixed.

**Comment by Unknwon** [comment]
Thanks, will tag the version ASAP.

**Comment by AT1012** [comment]
Thank you!

**Comment by Unknwon** [comment]
https://github.com/go-ini/ini/releases/tag/v1.12.0

**Pull request merged**